lib/rubocop/cop/rails/time_zone.rb

```diff
@@ -0,0 +1,93 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks for the use of Time methods without zone.
+      #
+      # Built on top of Ruby and Rails style guide (https://github.com/bbatsov/rails-style-guide#time)
+      # and the article http://danilenko.org/2012/7/6/rails_timezones/ .
+      #
+      # Two styles are supported for this cop. When EnforcedStyle is 'always'
+      # then only use of Time.zone is allowed.
+      #
+      # When EnforcedStyle is 'acceptable' then it's also allowed
+      # to use Time.in_time_zone.
+      #
+      # @example
+      #   # always offense
+      #   Time.now
+      #   Time.parse('2015-03-02 19:05:37')
+      #
+      #   # no offense
+      #   Time.zone.now
+      #   Time.zone.parse('2015-03-02 19:05:37')
+      #
+      #   # no offense only if style is 'acceptable'
+      #   DateTime.strptime(str, "%Y-%m-%d %H:%M %Z").in_time_zone
+      #   Time.at(timestamp).in_time_zone
+      class TimeZone < Cop
+        include ConfigurableEnforcedStyle
+
+        MSG = 'Do not use `%s` without zone. Use `%s` instead.'
+
+        TIMECLASS = [:Time, :DateTime]
+
+        DANGER_METHODS = [:now, :local, :new, :strptime, :parse, :at]
+
+        def on_const(node)
+          _, klass = *node.children
```

Comment:
Please, use more descriptive names instead of `_` - e.g. `_module`. This makes it easier for people to understand what's the structure where deconstructing. Btw, you can deconstruct `node` directly - no need to invoke `children`.